src/store/modules/product/ProductState.ts

```diff
@@ -1,7 +1,11 @@
 export default interface ProductState {
+  cached: any;
```

Comment:
We could revert changes done for cached products if not in use

Reply:
Reverted changes related to cached products.